fastify.js

```diff
@@ -765,3 +781,15 @@ function buildRoutePrefix (instancePrefix, pluginPrefix) {
 }
 
 module.exports = build
+module.exports.Get = Get
+module.exports.Post = Post
+module.exports.Delete = Delete
+module.exports.Head = Head
+module.exports.Put = Put
+module.exports.Options = Options
+module.exports.Patch = Patch
+module.exports.All = All
+module.exports.Hook = Hook
+module.exports.DecorateRequest = DecorateRequest
+module.exports.DecorateReply = DecorateReply
+module.exports.DecorateInstance = DecorateInstance
```

Comment:
Would it be possible to use these with JavaScript as well?

Reply:
The decorators can be used with babel.
Using them without typescript or babel should be possible, but it is not very practical.